.travis.yml

```diff
@@ -100,6 +100,8 @@ script:
         bash tests/check_sanity.sh;
     fi
 
+  - php -d memory_limit=-1 ./vendor/bin/phpstan analyse src
```

Comment:
This will make phpstan run several times (see https://travis-ci.com/PrestaShop/PrestaShop/builds/145533643). Consider creating a Github action instead.